includes/admin/post-types/meta-boxes/class.llms.meta.box.award.engagement.submit.php

```diff
@@ -60,7 +60,9 @@ public function configure() {
 		$this->priority = 'high';
 
 		// Remove wp core post submit meta box.
-		remove_meta_box( 'submitdiv', $this->screens, 'side' );
+		if ( function_exists( 'remove_meta_box' ) ) { // When this class is instantiated at `wp` to show an error message, the message `remove_meta_box()` is not available.
+			remove_meta_box( 'submitdiv', $this->screens, 'side' );
+		}
```

Comment:
I noticed this earlier but if there were no problems I didn't care, however, we have a "pattern" of removing metaboxes here already which won't be called out of it's intended context: https://github.com/gocodebox/lifterlms/blob/e6a8846161a2d62e5ee95ad8238ed3b81314d829/includes/admin/post-types/class.llms.meta.boxes.php#L162

Reply:
oh yeah I saw that a couple of days ago. I'm going to use that then.

Reply:
Oh better, I just don't need the remove at all.
using the same `id` overrides the wp core one.